datadog/resource_datadog_dashboard_list_test.go

```diff
@@ -106,21 +107,22 @@ func TestDatadogDashListImport(t *testing.T) {
 func testAccCheckDatadogDashListDestroy(accProvider *schema.Provider) resource.TestCheckFunc {
 	return func(s *terraform.State) error {
 		providerConf := accProvider.Meta().(*ProviderConfiguration)
-		client := providerConf.CommunityClient
+		datadogClientV1 := providerConf.DatadogClientV1
+		authV1 := providerConf.AuthV1
 
-		return datadogDashListDestroyHelper(s, client)
+		return datadogDashListDestroyHelper(s, authV1, datadogClientV1)
 	}
 }
 
-func datadogDashListDestroyHelper(s *terraform.State, client *datadog.Client) error {
+func datadogDashListDestroyHelper(s *terraform.State, authV1 context.Context, datadogClientV1 *datadogV1.APIClient) error {
 	for _, r := range s.RootModule().Resources {
 		if !strings.Contains(r.Primary.Attributes["name"], "List") {
 			continue
 		}
 		id, _ := strconv.Atoi(r.Primary.ID)
-		_, errList := client.GetDashboardList(id)
+		_, _, errList := datadogClientV1.DashboardListsApi.GetDashboardList(authV1, int64(id)).Execute()
 		if errList != nil {
-			if strings.Contains(errList.Error(), "not found") {
+			if strings.Contains(errList.Error(), "Not Found") {
```

Comment:
Why did it change?

Reply:
Previously the community client returned the error object as well: `404 Not Found {"errors": "...{dash_id} not found"}`. Where as the go client returns `404 Not Found` alone. Which is why the following never failed when changing from communityclient -> datadog go client: https://github.com/skarimo/terraform-provider-datadog/blob/master/datadog/resource_datadog_dashboard_list.go#L171